simpleclient_hibernate/src/main/java/io/prometheus/client/hibernate/HibernateStatisticsCollector.java

```diff
@@ -0,0 +1,282 @@
+
+        createCounter(
+            "hibernate_entity_delete_total",
+            "Get global number of entity deletes",
+            statistics.getEntityDeleteCount()
+        ),
+        createCounter(
+            "hibernate_entity_insert_total",
+            "Get global number of entity inserts",
+            statistics.getEntityInsertCount()
+        ),
+        createCounter(
+            "hibernate_entity_load_total",
+            "Get global number of entity loads",
+            statistics.getEntityLoadCount()
+        ),
+        createCounter(
+            "hibernate_entity_fetch_total",
+            "Get global number of entity fetches",
+            statistics.getEntityFetchCount()
+        ),
+        createCounter(
+            "hibernate_entity_update_total",
+            "Get global number of entity updates",
+            statistics.getEntityUpdateCount()
+        ),
+
+        createCounter(
+            "hibernate_collection_load_total",
+            "Global number of collections loaded",
+            statistics.getCollectionLoadCount()
+        ),
+        createCounter(
+            "hibernate_collection_fetch_total",
+            "Global number of collections fetched",
+            statistics.getCollectionFetchCount()
+        ),
+        createCounter(
+            "hibernate_collection_update_total",
+            "Global number of collections updated",
+            statistics.getCollectionUpdateCount()
+        ),
+        createCounter(
+            "hibernate_collection_remove_total",
+            "Global number of collections removed",
+            statistics.getCollectionRemoveCount()
+        ),
+        createCounter(
+            "hibernate_collection_recreate_total",
+            "Global number of collections recreated",
+            statistics.getCollectionRecreateCount()
+        )
+
+    );
+  }
+
+  private List<MetricFamilySamples> getQueryExecutionMetrics() {
+    return Arrays.asList(
+
+        createCounter(
+            "hibernate_query_execution_total",
+            "Get global number of executed queries",
+            statistics.getQueryExecutionCount()
+        ),
+
+        createGauge(
+            "hibernate_query_execution_seconds_max",
+            "Get the time in milliseconds of the slowest query.",
+            statistics.getQueryExecutionMaxTime() / 1000.0
+        ),
+
+        createCounter(
+            "hibernate_natural_id_query_execution_total",
+            "Get the global number of naturalId queries executed against the database",
+            statistics.getNaturalIdQueryExecutionCount()
+        ),
+
+        createGauge(
+            "hibernate_natural_id_query_execution_seconds_max",
+            "Get the global maximum query time for naturalId queries executed against the database",
+            statistics.getNaturalIdQueryExecutionMaxTime() / 1000.0
+        )
+
+    );
+  }
+
+  private CounterMetricFamily createCounter(String name, String help, double value) {
```

Comment:
Ok! I wasn't sure whether there will be some common label for all metrics. That's why I added this method. But I'll drop it.